vars/fortifyScan.groovy

```diff
@@ -0,0 +1,23 @@
+#!groovy
+import uk.gov.hmcts.contino.Builder
+import uk.gov.hmcts.contino.PipelineCallbacksRunner
+import uk.gov.hmcts.contino.AppPipelineConfig
+
+def call(params) {
+  PipelineCallbacksRunner pcr = params.pipelineCallbacksRunner
+  AppPipelineConfig config = params.appPipelineConfig
+  Builder builder = params.builder
+
+  def environment = params.environment
+  def product = params.product
+
+  if (config.fortifyScan) {
+    stageWithAgent("Fortify Scan", product) {
+      withFortifySecrets(product, environment) {
+        pcr.callAround('fortifyscan') {
```

Comment:
@timja  - Is the syntactic convention to use an all-lower case key without a space, in this line? eg `fortifyscan`

Reply:
no space, a dash can be used though

```suggestion
        pcr.callAround('fortify-scan') {
```